dist/bundle.js

```diff
@@ -425,7 +425,7 @@ var lodash = createCommonjsModule(function (module, exports) {
   var root = freeGlobal || freeSelf || Function('return this')();
 
   /** Detect free variable `exports`. */
-  var freeExports = exports && !exports.nodeType && exports;
+  var freeExports =  exports && !exports.nodeType && exports;
```

Comment:
hmm 🤔

Reply:
Not sure what that's all about... this file is generated.